tests/congruence_tests/test_delete_points.py

```
@@ -1,97 +1,103 @@
+    compare_collections(local_client, remote_client, NUM_VECTORS, attrs=("points_count",))
 
     compare_client_results(
         local_client,
         remote_client,
-        lambda c: c.search(COLLECTION_NAME, query_vector=NamedVector(name="image", vector=vector)),
+        lambda c: c.search(collection_name, query_vector=NamedVector(name="image", vector=vector)),
     )
 
-    #delete non-existent points
+    # delete non-existent points
     local_client.delete(COLLECTION_NAME, found_ids)
```

Ruff (0.8.2)

Undefined name `COLLECTION_NAME`

(F821)